StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentSheetVerticalViewController.swift

```diff
@@ -174,35 +175,29 @@ class PaymentSheetVerticalViewController: UIViewController, FlowControllerViewCo
         if let paymentMethodFormViewController {
             remove(childViewController: paymentMethodFormViewController)
         }
-
-        // If we'd only show one PM in the vertical list, and it collects user input, display the form instead of the payment method list.
         if shouldDisplayFormOnly, let paymentMethodType = loadResult.paymentMethodTypes.first {
+            // If we'd only show one PM in the vertical list, display the form instead of the payment method list.
```

Comment:
I think the old comment was better!

>         // If we'd only show one PM in the vertical list, and it collects user input, display the form instead of the payment method list.